src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import Panel, { PanelProps } from './Panel';
+
+const panelContent = <p>{faker.lorem.words()}</p>;
+
+const panelHeader = <p>{faker.lorem.words()}</p>;
+
+const onClose = jest.fn();
+
+const panelClassNames = {
+  panel: faker.lorem.word(),
+  header: faker.lorem.word(),
+  body: faker.lorem.word(),
+  closeButton: faker.lorem.word()
+};
+
+const defaultProps: PanelProps = {
+  header: panelHeader,
+  children: panelContent,
+  classNames: panelClassNames
+};
+
+const renderPanel = (props: PanelProps) => {
+  return <Panel {...props}></Panel>;
```

Comment:
Can this be a self closing component since it is empty?

Reply:
That's done